### PR TITLE
[YUNIKORN-1271] update people pages

### PR DESCRIPTION
**Pull request by wilfred-s**
Major PMC update as many people were missing
New committer

### CODE REVIEW

**Review by manirajv06 — APPROVED** [review]
looks good

**Review by yangwwei — APPROVED** [review]

**Review by pbacsko — APPROVED** [review]
+1

**Comment by steinsgateted** [comment]
Hi all,
I would like to add organization and timezone
Thank you.

**Comment by wilfred-s** [comment]
@steinsgateted please add it as a review comment on the change and we'll fix it.

File: i18n/zh-cn/docusaurus-plugin-content-pages/community/people.md [hunk 1]
```diff
@@ -28,38 +28,45 @@ YuniKorn 社区是开放的、多样化的和协作的。
...
-| Li Gao         | ligao    | Databricks | -8   |
-| Manikandan R   | mani     | Caastle    | +5.5 |
-| Peter Bacsko   | pbacsko  | Cloudera   | +1   |
-| Tingyao Huang  | tingyao  | TrendMicro | +8   |
-| Yu Teng Chen   | yuteng   | NTCU       | +8   |
+| 名称             | ApacheID      | 组织          | 时区 <br/>(UTC) |
+|----------------|---------------|-------------|---------------|
+| Chia-Ping Tsai | chia7712      | Is-Land     | +8            |
+| Li Gao         | ligao         | Databricks  | -8            |
+| Ted Lin        | steinsgateted |             |               |
```
Comment (steinsgateted) [inline i18n/zh-cn/docusaurus-plugin-content-pages/community/people.md:67]: NCKU,+8

File: src/pages/community/people.md [hunk 2]
```diff
@@ -29,41 +29,47 @@ by any entity. Project members share the responsibility to manage, and maintain
...
-| Li Gao         | ligao    | Databricks   | -8       |
-| Manikandan R   | mani     | Caastle      | +5.5     |
-| Peter Bacsko   | pbacsko  | Cloudera     | +1       |
-| Tingyao Huang  | tingyao  | TrendMicro   | +8       |
-| Yu Teng Chen   | yuteng   | NTCU         | +8       |
+| Name           | ApacheID      | Organization | Timezone <br/>(UTC) |
+|----------------|---------------|--------------|---------------------|
+| Chia-Ping Tsai | chia7712      | Is-Land      | +8                  |
+| Li Gao         | ligao         | Databricks   | -8                  |
+| Ted Lin        | steinsgateted |              |                     |
```
Comment (steinsgateted) [inline src/pages/community/people.md:70]: NCKU,+8

**Review by steinsgateted — CHANGES_REQUESTED** [review]

File: src/pages/community/people.md (hunk 2 above)
Comment (wilfred-s) [reply src/pages/community/people.md]: added

File: i18n/zh-cn/docusaurus-plugin-content-pages/community/people.md (hunk 1 above)
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-pages/community/people.md]: added

**Comment by steinsgateted** [comment]
Thank you.

**Review by steinsgateted — APPROVED** [review]

**Pull request closed without merging**